django/db/backends/postgresql/schema.py

```diff
@@ -58,6 +58,44 @@ def quote_value(self, value):
         # getquoted() returns a quoted bytestring of the adapted value.
         return adapted.getquoted().decode()
 
+    def quote_comment_value(self, value):
+        return self.quote_value(value)
+
```

Comment:
```suggestion
```

It turns out that we don't need to have specific behaviour for PostgreSQL which treats `''` and `NULL` the same.

See [this fiddle](https://dbfiddle.uk/8SiuoziM) for details.